src/styles/stopwatch.scss

```diff
@@ -0,0 +1,67 @@
+.stopwatch {
+  position: relative;
+  width: $stopwatch-width;
+  height: $stopwatch-height;
+  border: $border-width dotted $border-color;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    position: absolute;
+    height: $minutes-hand-height;
+    width: $hand-width-minutes;
+    background-color: $minutes-hand-color;
+    bottom: 50%;
+    left: 50%;
+    transform: translateX(-50%);
+    transform-origin: bottom center;
```

Comment:
You've used the `transform` property twice for the `.stopwatch__minutes-hand` class, which will result in the second declaration (transform-origin) overriding the first one (translateX(-50%)). To fix this, you should combine both transform functions into a single property, for example: `transform: translateX(-50%) translateY(-50%);`.